src/operations/aur_install/common.rs

```diff
@@ -196,25 +201,32 @@ async fn build_package(
         });
     }
 
-    let mut packages = MakePkgBuilder::package_list(build_path).await?;
-    let match_version = ctx
-        .package
-        .metadata
-        .version
-        .rsplit_once('_')
-        .map(|v| v.0)
-        .unwrap_or(&ctx.package.metadata.version);
-    let match_name = format!("{pkg_name}-{match_version}");
-    tracing::debug!("Match name {match_name}");
-    packages.retain(|name| {
-        name.file_name()
-            .and_then(|n| n.to_str())
-            .unwrap()
-            .starts_with(&match_name)
-    });
-    tracing::debug!("Archives: {packages:?}");
+    let archives = MakePkgBuilder::package_list(build_path).await?;
+    tracing::debug!("Archives: {archives:?}");
+
+    let mut pkgs_produced: HashMap<String, PathBuf> = HashMap::new();
+    let alpm = crate::internal::alpm::get_handler()?;
+
+    for ar in archives {
+        let pkg = alpm
```

Comment:
Can you encapsulate alpm stuff in a different module?

Reply:
I'll split it off into the already existing `alpm` module we have

Reply:
This should be good :+1: